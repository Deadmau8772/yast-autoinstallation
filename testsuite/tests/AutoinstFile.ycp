{

include "testsuite.ycp";
import "AutoinstFile.ycp";

DUMP("Import");
TEST(``(AutoinstFile::Import(["A"])), [], nil);

DUMP("Export");
TEST(``(AutoinstFile::Export()), [], nil);

}
