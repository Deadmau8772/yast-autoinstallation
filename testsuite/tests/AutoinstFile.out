Dump	Import
Return	true
Dump	Export
Return	["A"]
